runtime/moonbase/src/lib.rs

```diff
@@ -158,8 +158,7 @@ pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(WEIGHT_REF_TIME_PER_
 	.saturating_div(2)
```

Comment:
What about this TODO? Is it still applicable?

Reply:
I guess it's related to the comment in the PR description explaining that increasing the CPU time per block breaks many assumptions, right?

Reply:
Yes, we will apply this change in another PR